### PR TITLE
add footer and fix height to 100%, margin and padding to 0

### PR DESCRIPTION
**Pull request by pisethchhom**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,107 @@
...
 <body>
-    <h1>Hello freeCodeCamp Jackson, MS</h1>
-    <p>Today we're learing git and GitHub</p>
+    <header class="hero">
+        <h1>Hello freeCodeCamp Jackson, MS</h1>
+        <p>Today we're learning git and GitHub</p>
+        <img src="./assets/HF-full-logo-b05d5eb32b3f3ecc9b2240526104cf4da3187b8b61963dd9042fdc2536e4a76c.svg" alt="hacktoberfest 2020 logo" class="fest_img">
+    </header>
+
+    <div style="height: 100%;">
```
Comment (techieeliot) [inline index.html]: Do you mind adding a class and adding styling into the class? @pisethchhom

File: index.html [hunk 2]
```diff
@@ -1,12 +1,107 @@
 <!DOCTYPE html>
-<html lang="en">
+<!-- newly added 
+    * add height 100%
+    * reset padding 0
+    * reset margin 0
+-->
+<html lang="en" style="height: 100%; padding: 0; margin: 0;">
```
Comment (techieeliot) [inline index.html]: The style="..." content in the html element is not necessary. Please remove it @pisethchhom

File: index.html [hunk 3]
```diff
@@ -1,12 +1,107 @@
...
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Hacktoberfest and GitHub</title>
+    <style>
+
+        body{
+            background-color: #193D5D;
+            color: #ffffff;
+            width: 100%;
+            /* newly added
```
Comment (techieeliot) [inline index.html]: Don't really need this commented area @pisethchhom

**Review by techieeliot — CHANGES_REQUESTED** [review]
Please fix the minor changes and send those in a comment. Good job so far.

**Comment by techieeliot** [comment]
Run  the following commands

```
git remote add upstream git@github.com:techieeliot/hacktoberfest-demo.git
git fetch upstream main
git rebase -i upstream/main
```

Save your commit at the top and delete these extra commits below:
Screen Shot 2020-10-06 at 4 13 39 PM

Let me know if that makes sense. You should only have one commit.

File: index.html (hunk 2 above)
Comment (pisethchhom) [reply index.html]: the style height: 100% make the page full screen and footer appear in the bottom if I remove it the footer will not always appear in the bottom

**Review by techieeliot — APPROVED** [review]
👍 LGTM!

**Pull request merged**